### ISSUE TITLE
Elements in DynamicDialog change size after opening in Firefox

### ISSUE THREAD

**Issue by alekk84** [primefaces/primeng#8418]
```
[x] bug report
```
Elements in DynamicDialog change size after opening in Firefox

**Current behavior**
Firefox: After open DynamicDialog ( immediately or sometimes after moving mouse) elements are slightly change size.
![firefox](https://user-images.githubusercontent.com/25245657/71445993-de9d0500-272f-11ea-9c9b-76eeec107e8a.jpg)

Chrome: After opening DynamicDialog all items are displayed normally.
![chrome](https://user-images.githubusercontent.com/25245657/71445994-e2308c00-272f-11ea-90e2-938422b0f3cd.jpg)

**Minimal reproduction of the problem with instructions**
https://stackblitz.com/edit/angular-dynamic-dialog-bug

**Environment:**
* **Angular version:** 8.2.14

* **PrimeNG version:** 8.1.1

* **Browser:** [Firefox 71]

**Comment by alekk84** [comment]
When i set `showHeader: false` for DinamicDialogConfig this effect became apparent only after mouse hover over the control

![rgLleu3E7e](https://user-images.githubusercontent.com/25245657/71474878-a3add680-27ee-11ea-925b-825dd267d9fa.gif)

**Comment by cagataycivici** [comment]
Unable to replicate with PrimeNG 9.0.5 which has a new Dialog implementation, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
